packages/cli/generators/project/templates/.eslintrc.js.ejs

```diff
@@ -0,0 +1,50 @@
+<% if (project.loopbackBuild) { -%>
+module.exports = {
+  extends: '@loopback/eslint-config',
+};
+<% } else { -%>
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: ['eslint-plugin', '@typescript-eslint'],
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+  rules: {
```

Comment:
Please don't duplicate the content of `@loopback/eslint-config` here, it will quickly get out of sync.

IMO, we should always extend from `@loopback/eslint-config`, it's a shared config that's can be used independently from `@loopback/build`.

Reply:
+1.